internal/website/content/pages/secrets.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Secrets"
+---
+
+Package secrets provides an easy and portable way to encrypt and decrypt
+messages.
```

Comment:
Can we add a bit more details here? From reading just this one line it's not clear why we need a portable cloud API for this as opposed to using Go's bultin crypto packages

Reply:
Added some, PTAL.